sync/class.jetpack-sync-defaults.php

```diff
@@ -202,6 +201,76 @@ class Jetpack_Sync_Defaults {
 		'_wp_attached_file',
 	);
 
+	static $default_whitelist_post_meta_keys = array(
+	//	'_edit_last',
+	//	'_edit_lock',
+		'_feedback_akismet_values',
+		'_feedback_email',
+		'_feedback_extra_fields',
+		'_g_feedback_shortcode',
+		'_jetpack_author',
+		'_jetpack_author_email',
+		'_jetpack_dont_email_post_to_subs',
+		'_jetpack_post_author_external_id',
+		'_jetpack_post_thumbnail',
+		'_jetpack_related_posts_cache_()',
```

Comment:
Why the extra brackets here? As far as I can tell from the code, the real meta here is `_jetpack_related_posts_cache`